tests/spark_like_test.py

```diff
@@ -943,3 +954,175 @@ def test_left_join_overlapping_column(pyspark_constructor: Constructor) -> None:
+def test_is_duplicated(pyspark_constructor: Constructor) -> None:
+    data = {"a": [1, 1, 2], "b": [1, 2, 3], "level_0": [0, 1, 2]}
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(
+        a=nw.col("a").is_duplicated(),
+        b=nw.col("b").is_duplicated(),
+        level_0=nw.col("level_0"),
+    ).sort("level_0")
+    expected = {
+        "a": [True, True, False],
+        "b": [False, False, False],
+        "level_0": [0, 1, 2],
+    }
+    assert_equal_data(result, expected)
+
+
+def test_is_nan(pyspark_constructor: Constructor) -> None:
+    data = {"a": [1.0, float("nan"), 2.0, None, 3.0]}
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(nan=nw.col("a").is_nan())
+    expected = {"nan": [False, False, False, False, False]}
+    assert_equal_data(result, expected)
+
+
+# copied from tests/expr_and_series/is_finite_test.py
+def test_is_finite(pyspark_constructor: Constructor) -> None:
+    data = {"a": [float("nan"), float("inf"), 2.0, None]}
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(finite=nw.col("a").is_finite())
+    expected = {"finite": [False, False, True, False]}
+    assert_equal_data(result, expected)
+
+
+def test_is_in(pyspark_constructor: Constructor) -> None:
+    data = {"a": [1, 2, 3, 4, 5]}
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(in_list=nw.col("a").is_in([2, 4]))
+    expected = {"in_list": [False, True, False, True, False]}
+    assert_equal_data(result, expected)
+
+
+# copied from tests/expr_and_series/is_unique_test.py
+def test_is_unique(pyspark_constructor: Constructor) -> None:
+    data = {
+        "a": [1, 1, 2],
+        "b": [1, 2, 3],
+        "level_0": [0, 1, 2],
+    }
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(
+        a=nw.col("a").is_unique(),
+        b=nw.col("b").is_unique(),
+        level_0=nw.col("level_0"),
+    ).sort("level_0")
+    expected = {
+        "a": [False, False, True],
+        "b": [True, True, True],
+        "level_0": [0, 1, 2],
+    }
+    assert_equal_data(result, expected)
+
+
+def test_len(pyspark_constructor: Constructor) -> None:
+    data = {"a": [1, 2, float("nan"), 4, None], "b": [None, 3, None, 5, None]}
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(
+        a=nw.col("a").len(),
+        b=nw.col("b").len(),
+    )
+    expected = {"a": [5], "b": [5]}
+    assert_equal_data(result, expected)
+
+
+# copied from tests/expr_and_series/n_unique_test.py
+def test_n_unique(pyspark_constructor: Constructor) -> None:
+    data = {
+        "a": [1.0, None, None, 3.0],
+        "b": [1.0, None, 4, 5.0],
+    }
+    df = nw.from_native(pyspark_constructor(data))
+    result = df.select(
+        a=nw.col("a").n_unique(),
+        b=nw.col("b").n_unique(),
+    )
+    expected = {"a": [2], "b": [3]}
+    assert_equal_data(result, expected)
```

Comment:
None and nan should also count toward `n_unique`: see [`polars.Expr.n_unique`](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.n_unique.html#polars.Expr.n_unique)